### PR TITLE
Fixes #437 Fix bug "stopIfNotFound" not passed to "getOutputValues"

### LINKED ISSUE #437: stopIfNotFound = FALSE does not work in getOutputValues

### PR DESCRIPTION
**Pull request by PavelBal**
Fix bug "stopIfNotFound" not passed to "getOutputValues"

### CODE REVIEW

File: R/utilities-simulation-results.R [hunk 1]
```diff
@@ -77,7 +79,7 @@ getOutputValues <- function(simulationResults,
 
 
   for (path in paths) {
-    quantity <- getQuantity(path, simulationResults$simulation)
+    quantity <- getQuantity(path, simulationResults$simulation, stopIfNotFound = stopIfNotFound)
```
Comment (PavelBal) [inline R/utilities-simulation-results.R:82]: This was failing if the quantity is not found.

File: tests/testthat/test-utilities-simulation-results.R [hunk 2]
```diff
@@ -13,7 +13,16 @@ NUMBER_OF_EXTRA_COLUMNS <- NUMBER_OF_STATIC_COLUMNS + NUMBER_OF_COVARIATES_COLUM
 context("getOutputValues")
 
 test_that("It throws an error when no valid simulation results are provided", {
-  expect_error(getOutputValues(sim, resultsPaths))
+  expect_error(getOutputValues(individualResults, "NoPath"))
```
Comment (PavelBal) [inline tests/testthat/test-utilities-simulation-results.R:16]: This test was wrongly implemented.

File: DESCRIPTION [hunk 3]
```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: ospsuite
 Title: R package to manipulate OSPSuite Models
-Version: 9.1.0
+Version: 9.1.0001
```
Comment (msevestre) [inline DESCRIPTION:4]: Do not change this. This will be updated at build time

Comment (PavelBal) [reply DESCRIPTION]: Done.

Comment (PavelBal) [reply DESCRIPTION]: @msevestre Do you have more remarks?

File: tests/testthat/test-utilities-simulation-results.R (hunk 2 above)
Comment (msevestre) [reply tests/testthat/test-utilities-simulation-results.R:16]: oh so it was throwing an error but because the call was BS. Lol. OK

**Review by msevestre — APPROVED** [review]

**Pull request merged**